src/core/Akka.Persistence.Tests/Akka.Persistence.Tests.csproj

```diff
@@ -8,6 +8,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\contrib\serializers\Akka.Serialization.Hyperion\Akka.Serialization.Hyperion.csproj" />
+    <ProjectReference Include="..\Akka.Persistence.TestKit.Xunit2\Akka.Persistence.TestKit.Xunit2.csproj" />
```

Comment:
Referenced the Akka.Persistence.TestKit so I could more easily simulate journal / snapshot store failures